druzhba/redshift.py

```diff
@@ -148,3 +152,64 @@ def create_index_table(index_schema, index_table):
             """
             )
             logger.info("Index table %s.%s created", index_schema, index_table)
+
+
+EXTRACT_MONITOR_CREATE_TABLE_TEMPLATE = """
+CREATE TABLE {schema}.{table} (
+    task_id VARCHAR(max) NOT NULL,
+    class_name VARCHAR(1024) NOT NULL,
+    task_date_params VARCHAR(1024),
+    task_other_params VARCHAR(max),
+    start_dt TIMESTAMP NOT NULL,
+    end_dt TIMESTAMP,
+    run_time_sec INTEGER,
+    manifest_path VARCHAR(max),
+    data_path VARCHAR(max),
+    output_exists BOOLEAN,
+    row_count INTEGER,
+    upload_size BIGINT,
+    exception VARCHAR(1024),
+    created_at TIMESTAMP DEFAULT getdate() NOT NULL
+)
+SORTKEY (created_at);
+"""
+
+LOAD_MONITOR_CREATE_TABLE_TEMPLATE = """
+CREATE TABLE {schema}.{table} (
+    task_id VARCHAR(max) NOT NULL,
+    class_name VARCHAR(1024) NOT NULL,
```

Comment:
Some of these columns names don't really make sense for Druzhba now and are a relic of an earlier architecture (task_*, class_name). 

For both monitors it would be helpful to have columns referencing the source datastore name, table, etc.